README.md

```diff
@@ -60,6 +60,15 @@ With `config.py` in place, launch jira-gource:
 When the script starts downloading the tickets, it will prompt the user
 for their JIRA password to authenticate with the JIRA server.
 
+**If you are using Jira Cloud:**
+
+- You will receive a 401 code if you try to just authenticate with a password. You can easily use an API token instead.
```

Comment:
Please wrap the text at 80 characters to make it more readable in raw format and in terminals.